tools/release-scripts/test.js

```diff
@@ -16,10 +9,10 @@ function lint() {
 }
 
 function testAndLint() {
-  return Promise.all([
-    test(),
-    lint()
-  ]);
+  console.log('Running: '.cyan + 'eslint and tests'.green);
+
+  return exec('npm run test')
+    .then(() => console.log('Completed: '.cyan + 'eslint and tests'.green));
```

Comment:
Here you are addressing a bug that I've known about with the release process where the tests rely on the build before they can run due to the factories that get generated during the build. In essence this means that the build will get run more than once during the release process. You can see that here: https://github.com/react-bootstrap/react-bootstrap/blob/master/package.json#L14